eras/byron/ledger/impl/test/Test/Options.hs

```diff
@@ -18,9 +20,13 @@ module Test.Options
   )
 where
 
-import Cardano.Prelude hiding (Option)
 import Hedgehog (Gen, Group (..), Property, PropertyT, TestLimit, withTests)
 import Hedgehog.Internal.Property (GroupName (..), PropertyName (..))
+#if __GLASGOW_HASKELL__ >= 900
+import Cardano.Prelude
+#else
+import Cardano.Prelude hiding (Option)
+#endif
```

Comment:
@teodanciu And here we re-introduce Cardano-prelude 🙈

Reply:
It was there before, I just had to add pre-processor directives to prevent a name clash with the new Haskell prelude.

Reply:
Oh, yea it was there. I don't deny that. It's just that we didn't fully apply the remove Cardano-prelude patch. But for the release/1.1.x branch, I still don't think it matters. This is just our jumping board for 9.2 compatibility until we have a release from ledger @ master, to push into network on node.